trivia-app/client/src/components/AnswerChoices.js

```diff
@@ -0,0 +1,22 @@
+const AnswerChoices = ({index,  option, handlerUserChoice }) => {
+  
+  /** Send user choice back to parent */
+  const handleOption = (e) => {
+    handlerUserChoice(e); 
+  }
+
+  return (
+    <div className="RadioButton">
+      <input
+        id={`option${index+1}`}
+        type="radio"
+        name="answer_choice"
+        value={index}
+        onChange={handleOption}
+      />
+      <label htmlFor={`option${index+1}`}>{option}</label>
```

Comment:
why do we need the +1? are we using index+1 to correspond to something, or can we just use plain `index`?